frontend/demo/src/api/webStore.ts

```diff
@@ -0,0 +1,68 @@
+import { AiryConfig } from "./airyConfig";
+import { getCookie, setCookie } from "./cookie";
+import { User } from "../model/User";
+
+export const storeDomainCookie = (key: string) => (token: string) => {
+  let domain = AiryConfig.TOP_DOMAIN;
+
+  // If we are on development or don't have access to {domain} we set it to the current domain
+  if (AiryConfig.NODE_ENV !== "production" || !document.domain.endsWith("")) {
+    domain = document.domain;
+  }
+
+  setCookie(key, token, domain);
+};
+
+export const setUserId = storeDomainCookie("user_id");
+export const setAuthToken = storeDomainCookie("auth_token");
+export const getUserId = () => getCookie("user_id");
+export const getAuthToken = () => getCookie("auth_token");
+
+export function storeUserData(data: User) {
+  if (data.token) {
+    localStorage.setItem("id", data.id);
+    localStorage.setItem("first_name", data.first_name);
+    localStorage.setItem("last_name", data.last_name);
+    localStorage.setItem("isAuthSuccess", JSON.stringify(true));
```

Comment:
we don't need to stringify as far as I understand

Reply:
localStorage needs a string, that's why the stringify @lucapette

Reply:
https://stackoverflow.com/questions/28926997/how-to-have-localstorage-value-of-true

Reply:
TIL thank you for explaining!